tools/src/main/scala/scala/scalanative/codegen/BitMatrix.scala

```diff
@@ -0,0 +1,26 @@
+package scala.scalanative
+
+private[scalanative] class BitMatrix private (
```

Comment:
This is a minimal version of our java BitSet implementation with fixed size and no bound checks.